versioned_docs/version-8.4/apis-tools/zeebe-api/gateway-service.md

```diff
@@ -1206,3 +1206,83 @@ Returned if:
 Returned if:
 
 - The deleted resource is a process definition, and there are running instances for this process definition.
+
+## `StreamActivatedJobs` RPC
+
+Opens a long living stream for the given job type, worker name, job timeout, and fetch variables. This will cause available
+jobs in the engine to be activated and pushed down this stream.
+
+See the [job worker's technical reference](/versioned_docs/version-8.4/components/concepts/job-workers.md) for more on this.
+
+### Input `StreamActivatedJobsRequest`
+
+```protobuf
+message StreamActivatedJobsRequest {
+  // the job type, as defined in the BPMN process (e.g. <zeebe:taskDefinition
+  // type="payment-service" />)
+  string type = 1;
+  // the name of the worker activating the jobs, mostly used for logging purposes
+  string worker = 2;
+  // a job returned after this call will not be activated by another call until the
+  // timeout (in ms) has been reached
+  int64 timeout = 3;
+  // a list of variables to fetch as the job variables; if empty, all visible variables at
+  // the time of activation for the scope of the job will be returned
+  repeated string fetchVariable = 5;
+  // a list of identifiers of tenants for which to stream jobs
+  repeated string tenantIds = 6;
+}
+```
+
+### Output: a stream of `ActivatedJob`
+
+```protobuf
+message ActivatedJob {
+  // the key, a unique identifier for the job
+  int64 key = 1;
+  // the type of the job (should match what was requested)
+  string type = 2;
+  // the job's process instance key
+  int64 processInstanceKey = 3;
+  // the bpmn process ID of the job process definition
+  string bpmnProcessId = 4;
+  // the version of the job process definition
+  int32 processDefinitionVersion = 5;
+  // the key of the job process definition
+  int64 processKey = 6;
```

Comment:
❌This variable listed as `processDefinitionKey` in the original proto definition. Let's align this with the original variable name.

Reply:
I copied this from the `ActivateJobs` RPC above, so I also aligned that one.